perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/UsersManagerMethod.java

```diff
@@ -478,16 +478,24 @@ public UserExtSource call(ApiCaller ac, Deserializer parms) throws PerunExceptio
 	 * Remove user's external sources.
 	 * @param user int User <code>id</code>
 	 * @param userExtSource int UserExtSource <code>id</code>
+	 * @param force boolean. If true, use force deletion.
```

Comment:
Please remove dot after boolean keyword, it's wrongly parsed by our docs generating tool.